regexes/bots.yml

```diff
@@ -3472,6 +3472,35 @@
   category: 'Service Agent'
   url: 'https://www.phpmyadmin.net/'
 
+- regex: 'Matomo/([\d+.]+)'
```

Comment:
@sgiehl will there be any conflicts in motamo itself, if we add this?

Reply:
Good question. I would say it shouldn't.